README.md

```diff
@@ -1,53 +1,41 @@
-# AnyCloud
-
-AnyCloud is a Lambda alternative for Node.js that works with multiple cloud providers. We’re building the features of Lambda while fixing the problems we have run into while using it. Namely:
+<div align="center">
+  <img src="./assets/anycloud.png" alt="drawing" width="180"/>
+  <h1>AnyCloud scales webservers in any cloud provider</h2>
+</div>
+<br/>
+
+Our aim is to easily allow developers run their webservers on any cloud provider while also providing a much better experience to alternative services offered in AWS (Lambda, Elastic Beanstalk, ECS/Fargate, EC2) or GCP (Cloud Functions, App Engine, Cloud Run, Compute Engine).
```

Comment:
With these two as example is ok or should we add Azure alternatives on this README?